src/components/Sum.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export const Sum = ({
+  a,
+  b,
+}) => (
```

Comment:
Try not to add unnecessary line-breaks.
```suggestion
export const Sum = ({ a, b }) => (
```